README.md

```diff
@@ -82,6 +82,19 @@ You can find [an example csv file for import to Hyrax](https://github.com/curati
 with a heading of `files`, and the location of the files should be specified via an
 environment variables called `IMPORT_PATH`. If `IMPORT_PATH` is not set, `HyraxRecordImporter` will look in `/opt/data` by default.
 
+## Testing
+
+To run Solr and Fedora for testing purposes: 
+
+`solr_wrapper --config spec/dummy/config/solr_wrapper_test.yml`
+`fcrepo_wrapper --config spec/dummy/config/fcrepo_wrapper_test.yml -p 8986`
```

Comment:
can we just put the fedora port in the config file?

Reply:
In fact, it looks like the port is already configured there   
https://github.com/curationexperts/zizia/blob/ca04881b2672a49ce424d9ed1e86c3f92440b325/spec/dummy/config/fcrepo_wrapper_test.yml#L2